pages/configuration.tsx

```diff
@@ -1,22 +1,41 @@
 import type { NextPage } from "next";
 import { useEffect, useState } from "react";
+import { AppBridgeState } from "@saleor/app-bridge";
 
+
+import { SALEOR_DOMAIN_HEADER } from "../constants";
 import { useFetchTwelveOrdersQuery } from "../generated/graphql";
 import useApp from "../hooks/useApp";
 
 const Configuration: NextPage = () => {
-  const [{ data }] = useFetchTwelveOrdersQuery();
   const app = useApp();
-  const [domain, setDomain] = useState("");
 
-  useEffect(() => setDomain(app?.getState()!.domain || ""), [app]);
+  const [appState, setAppState] = useState<AppBridgeState>();
```

Comment:
Why do we keep the state in another state? If `state.ready` is false we could eg. return progress bar or show a message that configuration is being accessed outside of dashboard.